.github/workflows/performance-tests.yml

```
@@ -10,7 +10,8 @@ on:
       - synchronize
     paths:
       - 'R/**'
-      - 'src/**'       
+      - 'src/**'
+      - '.ci/atime/tests.R'
```

```suggestion
      - '.ci/atime/**'
```

Seems more appropriate